lib/beatable_ai_player.rb

```diff
@@ -0,0 +1,18 @@
+require "player"
+
+class BeatableAIPlayer
+  include Player
+
+  def initialize(mark, display)
+    @mark = mark
+    @display = display
+  end
+
+  def get_next_move
+    move = rand(1..9)
+    display.announce_player_move(mark, move)
+    move
+  end
+
+  attr_reader :mark, :display
```

Comment:
accessors should be at the top if they're public. only move them to the bottom of the file if you're changing visibility.